db/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/PersisterImLoeserTest.kt

```diff
@@ -133,8 +152,9 @@ class PersisterImLoeserTest {
             bekreftOpplysninger = true
         )
 
-        private val arbeidstaker = PersonDato("Test person", null, innsending.identitetsnummer)
-        private val arbeidsgiver = PersonDato("Test person", null, innsending.identitetsnummer)
+        val arbeidsgiver = PersonDato("Gudrun Arbeidsgiver", null, innsending.identitetsnummer)
+        val arbeidstaker = PersonDato("Toril Arbeidstaker", null, "fnr-gudrun")
```

Comment:
fnr-toril plz

Reply:
I'll do you one better: https://github.com/navikt/helsearbeidsgiver-inntektsmelding/pull/326/commits/5ef810b88b634e3c020a7c6cc48b912ef6de7f1b